index.js

```diff
@@ -0,0 +1,76 @@
+import menu from './modules/menu.js';
+// eslint-disable-next-line import/no-unresolved
+import Book from './modules/BookStorage.js';
+import { DateTime } from './modules/luxon.js';
+import storebooks from './modules/storebooks.js';
+
+menu();
+class UI {
```

Comment:
* It is a [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md#:~:text=Refactor%20your%20functions%20to%20arrow%20functions.) to Refactor your functions to arrow functions, Kindly change all functions to arrow functions